tests/integration/security/util/reachability/context.go

```diff
@@ -98,18 +97,22 @@ func (rc *Context) Run(testCases []TestCase) {
 		{
 			PortName: "http",
 			Scheme:   scheme.HTTP,
+			Timeout:  time.Second * 2,
```

Comment:
Is this too short to be sure configuration are pushed?

Reply:
timeout is per request, the retry is independent of this

Reply:
and is 30s currently

Reply:
is 30s per req comes from testing framework? if so, why not just change the TF?

Reply:
30s is the default for retry.UntilSuccess
not sure what the default client call is. I'm still investigating all these settings to see what makes the most sense